src/components/structure/Card/Card.tsx

```diff
@@ -0,0 +1,42 @@
+import * as S from './Card.styles';
+
+export type CardProps = {
+  cardImage: string;
+  bookName: string;
+  bookAuthors: string[];
+  totalPages: string;
+  bookPublisher: string;
+  publicationDate: string;
+};
+
+export const Card = ({
```

Comment:
Component properties are not being generic as a presentation component should be.